src/main/java/com/icthh/xm/ms/timeline/config/ApplicationProperties.java

```diff
@@ -1,12 +1,14 @@
 package com.icthh.xm.ms.timeline.config;
 
-import java.util.ArrayList;
-import java.util.List;
+import com.icthh.xm.commons.lep.TenantScriptStorage;
 import lombok.Data;
 import lombok.Getter;
 import lombok.Setter;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+import java.util.ArrayList;
```

Comment:
sort import by a-z  without separate section for java imports

Reply:
fixed